fern/api/definition/apiLoad.yml

```diff
@@ -0,0 +1,38 @@
+imports:
+  commons: commons.yml
+types:
+  Api:
+    properties:
+      apiId: commons.ApiId
+      packages: list<Package>
+  Package:
+    properties:
+      packageId: commons.PackageId
+      name: string
+      endpoints: list<Endpoint>
+  Endpoint:
+    properties:
+      endpointId: commons.EndpointId
+      endpointName: string
+  Type:
+    properties:
+      typeId: commons.TypeId
+      typeName: string
+  Error:
+    properties:
+      errorId: commons.ErrorId
+      errorName: string
+      statusCode: string
```

Comment:
we should figure out if the api can be in "invalid" states during edit mode and if so we might unions for each of these: 
```
Endpoint: 
  properties: 
    endpointId: commons.EndpointId
    data: EndpointData
 EndpointData: 
   union: 
     valid: ValidEndpoint
     invalid: InvalidEndpoint
```

Reply:
lets discuss